### ISSUE TITLE
pager ack! 400 error

### ISSUE THREAD

**Issue by d4v3y0rk** [hubot-scripts/hubot-pager-me#57]
I am getting this error when I am doing a `Hubot pager ack!`

Everything works fine when I do a `Hubot pager ack <incident number>`

https://gist.github.com/d4v3y0rk/2edb0cab3e324e9ef05b

```
User: awesomeuser asked me to "botzo pager ack!" in Channel: web-ops
400
{"error":{"message":"Invalid Input Provided","code":2001,"errors":["Status must be a list of triggered, acknowledged, or resolved."]}}
[Fri Oct 30 2015 13:58:00 GMT+0000 (UTC)] ERROR TypeError: Cannot read property 'incidents' of null
  at /opt/botzo/node_modules/hubot-pager-me/src/scripts/pagerduty.coffee:670:7, <js>:810:23
  at /opt/botzo/node_modules/hubot-pager-me/src/scripts/pagerduty.coffee:511:9, <js>:601:16
  at IncomingMessage.<anonymous> (/opt/botzo/node_modules/hubot/node_modules/scoped-http-client/src/index.js:95:22)
  at IncomingMessage.emit (events.js:129:20)
  at _stream_readable.js:908:16
  at process._tickCallback (node.js:355:11)
```

**Comment by stephenyeargin** [comment]
May be related to #56 (/cc @codywilbourn) -- planning to ship a patch today if there aren't any objections.

**Comment by codywilbourn** [comment]
Yes this is the same issue I ran in to. `ack <num>` works, `ack` and `ack!` do not.

**Comment by stephenyeargin** [comment]
Fixed in `v2.1.7`
